superset-frontend/src/dashboard/components/SliceHeaderControls/SliceHeaderControls.test.tsx

```diff
@@ -226,6 +227,43 @@ test('Should not show export full CSV if report is not table', async () => {
   expect(screen.queryByText('Export to full .CSV')).not.toBeInTheDocument();
 });
 
+test('Export full Excel is under featureflag', async () => {
+  // @ts-ignore
+  global.featureFlags = {
+    [FeatureFlag.ALLOW_FULL_CSV_EXPORT]: false,
+  };
+  const props = createProps('table');
+  renderWrapper(props);
+  userEvent.hover(screen.getByText('Download'));
+  expect(await screen.findByText('Export to Excel')).toBeInTheDocument();
+  expect(screen.queryByText('Export to full Excel')).not.toBeInTheDocument();
+});
+
+test('Should "export full Excel"', async () => {
+  // @ts-ignore
+  global.featureFlags = {
+    [FeatureFlag.ALLOW_FULL_CSV_EXPORT]: true,
```

Comment:
General note (not complaining to the author of this fine PR!): I wish we would not have used `CSV` all over the codebase. In the future, let's be more generic, like `ALLOW_FULL_FILE_EXPORT` or similar.